ChaosMod/Effects/db/Player/PlayerWantedController.cpp

```diff
@@ -1,5 +1,30 @@
 #include <stdafx.h>
 
+static void OnStartSix()
+{
+	Player player = PLAYER_ID();
+
+	SET_FAKE_WANTED_LEVEL(6);	
+	
+	SET_PLAYER_WANTED_LEVEL(player, 5, false);
+	SET_PLAYER_WANTED_LEVEL_NOW(player, false);
+
+	g_pEffectDispatcher->OverrideEffectName(EFFECT_6_STARS, EFFECT_6_STARS);
+	WAIT(60000);
```

Comment:
Also, if it is going to be a timed effect use the OnStop and OnTick for handling the time. It also allows for other improvements as mentioned above :)

Reply:
Yup. I will surely make it a timed effect. OnStop should I remove the wanted level, or just give them a 5 Start instead?